.github/workflows/build.yml

```diff
@@ -105,152 +105,152 @@ jobs:
 
   build-clang-cl-i386:
     name: Clang-CL (i386)
-    runs-on: windows-latest
+    runs-on: windows-2016
     steps:
-    - name: Install packages
-      run: |
-        choco install ninja -y
-        choco install --x86 -y llvm
-    - name: Install Flex and Bison
-      run: |
-        curl -O https://svn.reactos.org/storage/vperevertkin/flexbison.7z
-        7z x flexbison.7z -O${{github.workspace}}\bin
-        echo "${{github.workspace}}\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-        echo "BISON_PKGDATADIR=${{github.workspace}}\bin\share\bison" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
-        echo "M4=${{github.workspace}}\bin\m4.exe" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
-    - name: Add CL to PATH
-      uses: ilammy/msvc-dev-cmd@v1
-      with:
-        arch: amd64_x86
-    - uses: actions/checkout@v2
-      with:
-        path: src
-    - name: Configure
-      run: |
-        mkdir build
-        cd build
-        $env:PATH = "${env:PROGRAMFILES(X86)}\llvm\bin;$env:PATH"
-        cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE:FILEPATH=toolchain-msvc.cmake -DARCH:STRING=i386 -DENABLE_ROSTESTS=1 -DENABLE_ROSAPPS=1 -DUSE_CLANG_CL:BOOL=1 ${{github.workspace}}\src
-    - name: Build
-      working-directory: ${{github.workspace}}\build
-      run: cmake --build .
-    - name: Generate ISOs
-      working-directory: ${{github.workspace}}\build
-      run: |
-        cmake --build . --target bootcd
-        cmake --build . --target livecd
+      - name: Install packages
+        run: |
+          choco install ninja -y
+          choco install --x86 -y llvm
+      - name: Install Flex and Bison
+        run: |
```

Comment:
what's the matter with all these whitespace changes?

Reply:
Ease comparing, copy-pasting, ...
At least, use the same indentation for Linux and Windows...

Reply:
I second that. YAML is ill-defined enough that there is no standard regarding whitespaces.
If the original maintainer decided for the current whitespace style, don't change that as part of a PR that is meant to change something else entirely.

Reply:
> Formatting should be left at the discretion of the maintainer(s) of the respective file.

I am not sure what your point is:
this file has self-inconsistencies and is not listed in CODEOWNERS.

> All you did here is just to create meaningless noise with such whitespace changes,

I argue that all this inconsistency does is to create pain when I work on this file, especially when comparing/editing both Linux and Windows builds.

> so please revert them and keep only the changes that are in the context with this PR.

Arguable, considering (worth) 8b8108ddeefd84d527a43f3eabab4a0134638636 for example.

--

> YAML is ill-defined enough that there is no standard regarding whitespaces.

"Agreed", but I want to fix ROS file, not debate YAML definition.

> If the original maintainer decided for the current whitespace style,

Initial (a0283ecea3c10d081320a6fab0821a03cda6b7f8) committer is @Extravert-ir.
What is the rationale for using different indentations for different builds?

> don't change that as part of a PR that is meant to change something else entirely.

By popular demand, I'll split that part to a separate PR...

Reply:
It is very simple:

1. You are not the maintainer of this file
2. You have been asked numerous times not to change all kinds of random stuff unrelated to the PR.
3. Every time we have more or less the same discussion

Should we just close every PR where you do this?

Reply:
> It is very simple:

I wish it would be.

>     1. You are not the maintainer of this file

Obviously, but, as often, you don't tell who is, nor answer the other questions...

>     2. You have been asked numerous times not to change all kinds of random stuff unrelated to the PR.

True, and the very _opposite_ too:
see, for examples,
https://github.com/reactos/reactos/pull/3066#issuecomment-715343541
where @Extravert-ir closed all my "splitted" TaskMgr PRs, asking for an all-in-one instead,
or
https://github.com/reactos/reactos/pull/3512#issuecomment-795594098
where @Extravert-ir asked to do more dev within an "unrelated" PR,
NB:
No special blame to @Extravert-ir here, only examples from file initial committer.

Also, I would debate whether formatting this file is actually unrelated to updating it.

>     3. Every time we have more or less the same discussion

Mostly, yes, because either way I try to submit my PRs, it seems there is often a (different) dev who disagree :-(

> Should we just close every PR where you do this?

Sadly, that often happens no matter what.
I still "wonder" why a contributor like me can't submit a simple PR with 2 separate commits like this one,
while a dev can commit a single commit with separate formatting changes merged with code changes...

Reply:
> > It is very simple:
> 
> I wish it would be.
> 
> > ```
> > 1. You are not the maintainer of this file
> > ```
> 
> Obviously, but, as often, you don't tell who is, nor answer the other questions...

As it's not listed, I use the magic of git to see who is most active in this file, and that appears to be @Extravert-ir 

> 
> > ```
> > 2. You have been asked numerous times not to change all kinds of random stuff unrelated to the PR.
> > ```
> 
> True, and the very _opposite_ too:
> see, for examples,
> [#3066 (comment)](https://github.com/reactos/reactos/pull/3066#issuecomment-715343541)
> where @Extravert-ir closed all my "splitted" TaskMgr PRs, asking for an all-in-one instead,
> or

I sincerely hope you understand the difference between multiple pr's to change a few bits here and there,
and multiple PR's where you re-format an entire file.

> [#3512 (comment)](https://github.com/reactos/reactos/pull/3512#issuecomment-795594098)
> where @Extravert-ir asked to do more dev within an "unrelated" PR,
> NB:
> No special blame to @Extravert-ir here, only examples from file initial committer.
> 
> Also, I would debate whether formatting this file is actually unrelated to updating it.
> 

Here you are moving files around, and his suggestion is that if there is a commit that touches all lines with no changes, it would make sense to do this other action as well, since otherwise there are always 2 commits to skip past while looking for the history.

> > ```
> > 3. Every time we have more or less the same discussion
> > ```
> 
> Mostly, yes, because either way I try to submit my PRs, it seems there is often a (different) dev who disagree :-(
> 

Yeah, most of us are not paid devs, most of us do this in their spare time, and there are no rules set in stone for a lot of things.
So depending on the component being changed, other people will chime in.

> > Should we just close every PR where you do this?
> 
> Sadly, that often happens no matter what.
> I still "wonder" why a contributor like me can't submit a simple PR with 2 separate commits like this one,
> while a dev can commit a single commit with separate formatting changes merged with code changes...

It is preferred that a dev does not do this either,
but a maintainer (or regular dev) of a certain part is more or less in charge of formatting code / keeping it tidy.

Reply:
> As it's not listed, I use the magic of git to see who is most active in this file, and that appears to be @Extravert-ir

Hum, why not simply update CODEOWNERS?
Also, I do remember I used to look at history (to pick reviewers if I remember correctly) and got complain that doing that was not appropriate...

> I sincerely hope you understand the difference between multiple pr's to change a few bits here and there,
> and multiple PR's where you re-format an entire file.

Yes and no, I am not sure what your point/example(s) is.
I mostly feel someone often finds some sort of (different/varying) reason to say 'no'...

> Here you are moving files around, and his suggestion is that if there is a commit that touches all lines with no changes, it would make sense to do this other action as well, since otherwise there are always 2 commits to skip past while looking for the history.

#3512: Well, my intent is the opposite, not moving and changing things at he same time.
History: Yes, but having separate commits ease sorting things out, like (regression) testing...

> Yeah, most of us are not paid devs, most of us do this in their spare time, and there are no rules set in stone for a lot of things.

I know...

> So depending on the component being changed, other people will chime in.

That is somewhat a problem.
For example with this PR, if Victor is in charge, why do other people comment without waiting for him first?
In other words, if different devs/modules use different rules, then why not make the laters more explicit, to everyone?

> It is preferred that a dev does not do this either,
> but a maintainer (or regular dev) of a certain part is more or less in charge of formatting code / keeping it tidy.

That is part of the issue: it is not always obvious why a dev commits as he does then why code can't be "fixed".
Here (#3580 now), why is whitespace style not uniform across the file (initial commit) and why can't it be improved to make (local) dev'ing easier?

Reply:
> why do other people comment without waiting for him first?

We are no machines?

Reply:
> We are no machines?

??